torch_xla/csrc/xla_sharding_util.cpp

```diff
@@ -47,6 +49,13 @@ std::vector<int64_t> TileAssignmentDimensions(
 
 }  // namespace
 
+using Handle = int64_t;
+using ShardingInfo = ShardingUtil::TensorShardingStore::ShardingInfo;
+
+std::mutex ShardingUtil::TensorShardingStore::mutex_;
+std::unordered_map<Handle, ShardingInfo>
+    ShardingUtil::TensorShardingStore::tensor_sharding_map_;
```

Comment:
Are these defined in the header already?

Reply:
Declared in the header, initialized here.